git/git-cheatsheet-de.md

```diff
@@ -35,6 +35,27 @@ git config --global core.eol lf
 
 ## Rebase und Konflikte
 
+### Interaktive Rebase
+
+Rebase ausführen: `git rebase -i HEAD~<Anzahl der Schritte>`
+
+/*
+Commands in File-Edit Mode:
+
+p, pick = Commit verwenden
+r, reword = Commit verwenden und Commit Log ändern
+e, edit = Commit verwendung und den Commit selbst ändern Log und Inhalt
+s, squash = Commit verwenden und mit dem vorhergehende Commit verschmelzen
+f, fixup = wie "squash" und Log verwerfen
+x, exec = shell commando ausführen - restliche zeile nach x, exec
+d, drop = Commit löschen
+*/
+
+Datei speichern und schließen
+
```

Comment:
Bitte in Markdown immer nur eine Leerzeile am Stück. Siehe dazu: http://www.cirosantilli.com/markdown-style-guide/#newlines

Reply:
Habe ich geändert.